editoast/schemas/src/rolling_stock.rs

```diff
@@ -110,7 +110,7 @@ pub struct RollingStock {
     #[schema(example = 15.0)]
     #[serde(default, with = "units::second::option")]
     pub raise_pantograph_time: Option<Time>,
-    pub supported_signaling_systems: RollingStockSupportedSignalingSystems,
+    pub supported_signaling_systems: Vec<RollingStockSupportedSignalingSystem>,
```

Comment:
Should it be a `HashSet` instead? Because I do not think we would like twice the same value in this list.


```suggestion
    pub supported_signaling_systems: HashSet<RollingStockSupportedSignalingSystem>,
```